themes/classic/assets/css/theme.rtlfix

```diff
@@ -0,0 +1,12 @@
+.rtl-flip:not(.rtl-no-flip), .material-icons:not(.rtl-no-flip) {
```

Comment:
Are you sure about the `.rtlfix` file? Why not a `.scss` which will be compiled?
And be careful with the indent, please